frontend/src/app/[locale]/(main)/dashboard/page.tsx

```diff
@@ -56,6 +56,15 @@ export default async function DashboardPage() {
         />
         <StatCard value={`${userStats.currentStreakDays}d`} label={t('userStats.currentStreak')} />
         <StatCard value={`${userStats.averageScore ?? 0}%`} label={tCommon('avgScore')} />
+        {/* Remaining sessions stat card for non-admin users */}
+        {userProfile.accountRole !== 'admin' &&
```

Comment:
This should be visible as an admin as well to prevent confusion. Just replace the actual number of remaining sessions with N/A since admins do not have a limit.